install/update_2.5_2.6.php

```diff
@@ -25,5 +25,23 @@ function plugin_formcreator_update_2_6(Migration $migration) {
       $DB->query($query) or plugin_formcreator_upgrade_error($migration);;
    }
 
+   // add location rule
+   $enum_location_rule = "'".implode("', '", array_keys(PluginFormcreatorTargetTicket::getEnumLocationRule()))."'";
+   if (!FieldExists('glpi_plugin_formcreator_targettickets', 'location_rule', false)) {
+      $query = "ALTER TABLE `glpi_plugin_formcreator_targettickets`
```

Comment:
use ``$migration->addField``

Reply:
As far as I know Migration does not know how to deal with ENUIM. That's why I keep this style for now.

If I'm wrong, please show me the trick

Reply:
addField/changeField call Migration::fieldFormat which a list of shortcut cases

and a default case to copy directly the type (ENUM ... NOT NULL... etc)

Reply:
done